sdk/core/core-http-client-xhr/src/index.browser.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export { createXhrHttpClient } from "./xhrHttpClient.js";
```

Comment:
Shouldn't this be `index-browser.mts`?

Reply:
yeah, this PR was created prior to ESM migration, I haven't applied the migration completely.